src/Artemis.Core/DefaultTypes/DataBindings/Modifiers/Colors/SKColorRotateHueModifierType.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using SkiaSharp;
+
+namespace Artemis.Core.DefaultTypes
+{
+    internal class SKColorRotateHueModifierType : DataBindingModifierType
+    {
+        public override IReadOnlyCollection<Type> CompatibleTypes => new List<Type> {typeof(SKColor)};
+        public override Type ParameterType => typeof(float);
+
+        public override string Name => "Rotate Hue by";
+        public override string Icon => "RotateRight";
+        public override string Description => "Rotates the hue of the color by the amount in degrees";
+
+        public override object Apply(object currentValue, object parameterValue)
+        {
+            ((SKColor) currentValue).ToHsl(out var h, out var s, out var l);
+
+            h += (float)parameterValue;
+
+            var a =  SKColor.FromHsl(h % 360, s, l);
```

Comment:
How does this method work? If you need the hue to be between 0 and 360 you need to check for negative values, if not the % 360 is not required.

Reply:
Values above 360 don't work. I did not test negative values because you can't type '-' in the ui. I'll have to test with a negative property in a data model.